src/opm/output/eclipse/AggregateWellData.cpp

```diff
@@ -243,6 +238,71 @@ namespace {
             return WMCtrlVal::WMCtlUnk;
         }
 
+        bool wellControlDefind(const Opm::data::Well& xw)
```

Comment:
Maybe add an `e` in defined?

Reply:
> Maybe add an `e` in defined?

:man_facepalming: 

Thanks!  I'll do just that.